src/SamplesApp/SamplesApp.UITests/TestFramework/ImageAssert.ExpectedPixels.cs

```diff
@@ -107,7 +107,7 @@ public ExpectedPixels Pixels(_Bitmap source, Rectangle rect)
 
 	public ExpectedPixels Pixels(_Bitmap source)
 	{
-		var colors = new Color[source.Width, source.Height];
+		var colors = new Color[source.Height, source.Width];
```

Comment:
Hummm, good catch

BTW this file was cloned recently, you should also fix it here https://github.com/unoplatform/uno/blob/master/src/SamplesApp/SamplesApp.UITests/TestFramework/ImageAssert.ExpectedPixels.cs#L110

Reply:
It was not cloned, I changed that after your comments :)

Reply:
Well ... it was changed: the file is now shared by the 2 tests engines, so dismiss my comment.